bibliopixel/drivers/SPI/interfaces.py

```diff
@@ -1,8 +1,13 @@
-import math
 import os
-from . channel_order import ChannelOrder
-from . driver_base import DriverBase
-from .. import log
+import math
+from enum import IntEnum
+from . import errors
+from ... import log
+
+
+def error(text):
```

Comment:
This needs to be a method on SpiBaseInterface which reports `self._dev` too.  When this malfunctions, the first thing I want to know is the path to the driver that failed.